src/VuetifyResource.vue

```diff
@@ -58,10 +58,10 @@
             v-model="snackbar.active"
         >
             {{ snackbar.text }}
-            <v-btn @click.native="snackbar.active = false" dark flat>{{lang('close')}}</v-btn>
+            <v-btn @click.native="snackbar.active = false" dark text>{{lang('close')}}</v-btn>
         </v-snackbar>
 
-        <v-layout row wrap>
```

Comment:
Was this necessary to apply the new version 2.0?

Reply:
Yep, new vuetify version has another grid system. We've used row's the wrong way.